components/atom/button/src/index.scss

```diff
@@ -0,0 +1,180 @@
+@import '~@schibstedspain/sui-theme/lib/settings';
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$bgc-button: $c-primary !default;
+$bgc-button-light: color-variation($bgc-button, 4) !default;
+$bgc-button-dark: color-variation($bgc-button, -1) !default;
+$bgc-button-light-contrast: color-variation($bgc-button, 3) !default;
+
+$sz-button-small: $p-base*4 !default;
+$fz-button-small: $fz-s !default;
+$icon-sz-button-small: $sz-icon-s !default;
+$icon-m-button-small: $m-s !default;
+$p-button-small: $p-m !default;
+
+$sz-button: $p-base*5 !default;
+$fz-button: $fz-m !default;
+$icon-sz-button: $sz-icon-s !default;
+$icon-m-button: $m-m !default;
+$p-button: $p-l !default;
+
+$sz-button-large: $p-base*6 !default;
+$fz-button-large: $fz-m !default;
+$icon-sz-button-large: $sz-icon-m !default;
+$icon-m-button-large: $m-m !default;
+$p-button-large: $p-l !default;
+
+// Set all color properties for text and icons
+@mixin button-foreground-color ($color) {
+  color: $color;
+
+
+  & svg path {
+    fill: $color;
+    stroke: $color;
+  }
+}
+// Sets active states when button is not disabled
+@mixin button-focused {
+  &:not(.sui-AtomButton--disabled) {
+    &.sui-AtomButton--focused,
+    &:active,
+    &:focus,
+    &:hover {
+      @content;
+    }
+  }
+}
+@mixin button-icon ($size, $margin) {
+  & .sui-AtomButton-leftIcon,
+  & .sui-AtomButton-rightIcon {
+    height: $size;
+    width: $size;
+  }
+
+  & .sui-AtomButton-leftIcon {
+    margin-right: $margin;
+  }
+
+  & .sui-AtomButton-rightIcon {
+    margin-left: $margin;
+  }
+}
+
+.sui-AtomButton {
+  @include reset-button;
+  @include button-icon($icon-sz-button, $icon-m-button);
+  border: 1px solid;
+  border-radius: $bdrs-m;
```

Comment:
I would put a border width variable

Reply:
Borders are not a global variable and here it's used only once.
Why would you use a variable ? For future maintenance ?